sklearn/cluster/optics_.py

```diff
@@ -85,41 +86,19 @@ class OPTICS(BaseEstimator, ClusterMixin):
     metric_params : dict, optional (default=None)
         Additional keyword arguments for the metric function.
 
-    maxima_ratio : float, optional (default=.75)
-        The maximum ratio we allow of average height of clusters on the
-        right and left to the local maxima in question. The higher the
-        ratio, the more generous the algorithm is to preserving local
-        minima, and the more cuts the resulting tree will have.
-
-    rejection_ratio : float, optional (default=.7)
-        Adjusts the fitness of the clustering. When the maxima_ratio is
-        exceeded, determine which of the clusters to the left and right to
-        reject based on rejection_ratio. Higher values will result in points
-        being more readily classified as noise; conversely, lower values will
-        result in more points being clustered.
-
-    similarity_threshold : float, optional (default=.4)
-        Used to check if nodes can be moved up one level, that is, if the
-        new cluster created is too "similar" to its parent, given the
-        similarity threshold. Similarity can be determined by 1) the size
-        of the new cluster relative to the size of the parent node or
-        2) the average of the reachability values of the new cluster
-        relative to the average of the reachability values of the parent
-        node. A lower value for the similarity threshold means less levels
-        in the tree.
-
-    significant_min : float, optional (default=.003)
-        Sets a lower threshold on how small a significant maxima can be.
+    cluster_method : string, optional (default='dbscan')
+        The extraction method used to extract clusters using the calculated
+        reachability and ordering. Possible values are "dbscan".
+
+    eps : float, optional (default=0.5)
+        The maximum distance between two samples for them to be considered
+        as in the same neighborhood. Used ony when `cluster_method='dbscan'`.
 
     min_cluster_size : int > 1 or float between 0 and 1 (default=0.005)
         Minimum number of samples in an OPTICS cluster, expressed as an
         absolute number or a fraction of the number of samples (rounded
         to be at least 2).
-
-    min_maxima_ratio : float, optional (default=.001)
-        Used to determine neighborhood size for minimum cluster membership.
-        Each local maxima should be a largest value in a neighborhood
-        of the `size min_maxima_ratio * len(X)` from left and right.
+        Used only when `extract_method='xi'`.
```

Comment:
The `xi` method is not yet merged. So that would be a dead parameter for now.

Reply:
Yes. This parameter was used in sqlnk, but since that extraction method is removed, this parameter is dead. I haven't removed it cause I'd need to put it back again in `xi`.

Reply:
Let's keep the PR clean?

Reply:
The result of this PR is not clean anyway
- the examples and the docs will be wrong
- it doesn't make sense to have optics w/o `xi`

So I don't think we need to be too picky here.

Reply:
You'll break Circle CI on master. At least remove these examples.

Reply:
@assiaben I'm removing the `OPTICS` example, could you take a note here (and a copy) and develop an example based on the `extractXi` PR based on the being removed example?

@qinhanmin2014 removed the example.